cmake/compiler_config.cmake

```diff
@@ -5,8 +5,9 @@ set(GNU_CLANG_FLAGS
     # we want to be as strict as possible
     -Wall
     $<$<BOOL:${BUILD_STRICT_MODE}>:-Werror>
+    $<$<CONFIG:DEBUG>:-fstack-check>
     # optimization
-    -msse3
+    -msse4.2
```

Comment:
These GNU_CLANG_FLAGS are used by both clang and gcc. Is this change okay for our Linux builds?

Reply:
Yes, but current Maya is building with gcc 9, whereas we are still using gcc 6.3.1.

Reply:
This flag has been there for at least in Feb. 8, 2019 in Maya and it was merged from Peptide so it was in Peptide before that.

Reply:
Okay thanks for checking.